boards/phytec/mimx8mm_phyboard_polis/doc/index.rst

```diff
@@ -1,4 +1,4 @@
-.. _mimx8mm_phyboard_polis:
+.. _mimx8mm_phyboard_polis/mimx8mm6/m4:
```

Comment:
??

Reply:
Caught in a find and replace while renaming cores